src/login/login.module.ts

```diff
@@ -5,10 +5,12 @@ import { Module } from "@nestjs/common";
 import { OtpModule } from "../otp/otp.module";
 import { TokenModule } from "../token/token.module";
 import { UserModule } from "../user/user.module";
+import { SendLoginOtpEmailController } from "./send-otp-email-login.controller";
+import {EventEmitterModule} from '@nestjs/event-emitter';
 
 @Module({
-  imports: [TokenModule, UserModule, OtpModule],
-  controllers: [LoginController, LoginOtpEmailController],
+  imports: [TokenModule, UserModule, OtpModule, EventEmitterModule.forRoot()],
```

Comment:
`EventEmitterModule.forRoot()` is defined in the `app` module. Let's remove this definition.